### ISSUE TITLE
Tooltip not shown if the line contains a comment

### ISSUE THREAD

**Issue by micthiesen** [Twixes/pypi-assistant#3]
At least I believe this is what's happening. For example, a tooltip is shown for every package in my requirements file except this one:
```
ipython==7.12.0  # Necessary for shell_plus access in review app and production environments
```

Thanks for this extension by the way; it's pretty handy!

**Comment by micthiesen** [comment]
I believe this output from "Log (Window)" is related:
```javascript
[2020-09-17 15:49:23.324] [renderer1] [error] Illegal argument, contents must be defined: Error: Illegal argument, contents must be defined
	at new j (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:509:321)
	at new j (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:497:187)
	at Object.<anonymous> (/Users/michael/.vscode/extensions/twixes.pypi-assistant-1.0.3/out/extension.js:66:20)
	at Generator.next (<anonymous>)
	at /Users/michael/.vscode/extensions/twixes.pypi-assistant-1.0.3/out/extension.js:8:71
	at new Promise (<anonymous>)
	at __awaiter (/Users/michael/.vscode/extensions/twixes.pypi-assistant-1.0.3/out/extension.js:4:12)
	at Object.provideHover (/Users/michael/.vscode/extensions/twixes.pypi-assistant-1.0.3/out/extension.js:63:12)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:714:312
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:52:373
	at new Promise (<anonymous>)
	at Object.t.asPromise (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:52:345)
	at C.provideHover (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:714:283)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:735:479
	at K._withAdapter (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:732:331)
	at K.$provideHover (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:735:457)
	at g._doInvokeHandler (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:860:464)
	at g._invokeHandler (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:860:156)
	at g._receiveRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:858:766)
	at g._receiveOneMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:857:623)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:855:691
	at l.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:475)
	at v.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:256:381)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1047:649
	at l.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:475)
	at v.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:256:381)
	at t.PersistentProtocol._receiveMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:261:451)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:258:489
	at l.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:475)
	at p.acceptChunk (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:253:851)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:253:203
	at Socket.t (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:263:54)
	at Socket.emit (events.js:223:5)
	at Socket.EventEmitter.emit (domain.js:475:20)
	at addChunk (_stream_readable.js:309:12)
	at readableAddChunk (_stream_readable.js:290:11)
	at Socket.Readable.push (_stream_readable.js:224:10)
	at Pipe.onStreamRead (internal/stream_base_commons.js:181:23)
```

**Comment by Twixes** [comment]
Thanks for raising this point! I should have gotten to it way sooner, but this is now fixed in 1.0.4 ([along with other regex imperfections](https://github.com/Twixes/pypi-assistant/commit/49faec117963e9e7b7c35834262a6158e3793109)).
